src/msw/textentry.cpp

```diff
@@ -1123,13 +1123,24 @@ bool wxTextEntry::MSWShouldPreProcessMessage(WXMSG* msg) const
                             case VK_DELETE:
                             case VK_HOME:
                             case VK_END:
+                            case VK_LEFT:
+                            case VK_RIGHT:
                                 return false;
                         }
                     }
                     else // Shift is pressed
                     {
-                        if ( vkey == VK_INSERT || vkey == VK_DELETE )
-                            return false;
+                        switch ( vkey )
+                        {
+                            case ' ':
```

Comment:
Does `Shift-Space` have some special meaning for the text control? AFAICS it doesn't do anything special in it, so why should we have it here?

Reply:
I assume that we wanted Shift-Space to result in a normal space character even when a parent window interprets Shift-Space as an accelerator. Apparently Ctrl-Space is fine to be interpreted as an accelerator.

Reply:
I agree, special handling for `Shift-Space` was a mistake. I've removed it.